doc/.feature-template.md

```diff
@@ -21,7 +21,7 @@
 
 ## Using YourFeature with Uno
 
-* Add any steps required to use the feature that _aren't_ covered by Microsoft's UWP documentation, i.e. because they're Uno-specific. If there are none, you can safely remove this whole section.
+* Add any steps required to use the feature that _aren't_ covered by Microsoft's WinUI documentation, i.e. because they're Uno-specific. If there are none, you can safely remove this whole section.
```

Comment:
```suggestion
* Add any steps required to use the feature that _aren't_ covered by Microsoft's WinUI documentation, i.e., because they're Uno-specific. If there are none, you can safely remove this whole section.
```